runtime/rasdump/TextFileStream.cpp

```diff
@@ -87,7 +97,7 @@ TextFileStream::close(void)
 /* Methods for getting the object's status */
 bool TextFileStream::isOpen(void) const
 {
-	return _FileHandle != -1;
+	return _IsOpen;
```

Comment:
I'm not sure this is appropriate. When writing to stdout/stderr, `javadump.cpp` will report an error:
> J9NLS_DMP_NO_CREATE=Aborting: Cannot create file (%s)

We may need to expand the API of this class to avoid inappropriate diagnostics.

Reply:
I see. So should I revert this change, then perhaps create another issue for IsOpen()?

Reply:
I think it makes sense to address the javadump behavior in this PR (there is no issue without these changes).